donate/templates/payment/card.html

```diff
@@ -15,24 +15,26 @@
                 <div class="notification notification--neutral">
                     <div class="notification__container">
                         <div class="notification__message">
-                            {% trans "You are donating" %} {{ currency_info.symbol }} {{ form.amount.value|floatformat:"-2" }}
+                            {% trans "You are donating" %} {{ currency_info.symbol }} <span class="js-donation-value">{{ form.amount.value|floatformat:"-2" }}</span>
                         </div>
                         <div class="notification__action">
-                            <a href="#donate__amount-edit" data-amount-toggle>{% trans "Change amount" %}</a>
+                            <a href="#donate__amount-edit" class="donation-update__toggle" data-amount-toggle>{% trans "Change amount" %}</a>
                         </div>
                     </div>
                 </div>
 
                 <div class="donation-update" data-amount-actions>
-                    <div class="donation-update__container">
-                        <div class="form-item">
-                            <label for="donation-update">{{ currency_info.symbol }}</label>
-                            <input type="number" name="donation-update" value="{{ form.amount.value }}" step="1" min="1">
+                    <form class="donation-update__container">
+                        <div class="donation-update__form-item form-item">
+                            <label class="donation-update__label form-item__label" for="donation-update">{% trans "Donation Amount" %}</label>
+                            <div class="donation-update__currency-symbol">{{ currency_info.symbol }}</div>
+                            <input class="donation-update__input form-item__input" id="js-update-donation-value" type="number" name="donation-update" value="{{ form.amount.value }}" step="1" min="1">
                         </div>
                         <div class="form-item">
-                            <button class="button button--primary">{% trans 'Confirm' %}</button>
+                            <button class="donation-update__action button button--primary button--large" id="js-update-button">{% trans 'Confirm' %}</button>
```

Comment:
Could we update `id_amount` (rendered in line 66 of this file) instead?

Reply:
Done